src/utils_cmd_putval.c

```diff
@@ -35,6 +35,15 @@
  * private helper functions
  */
 
+static int is_quoted(const char *str, size_t len) {
+  if (len < 2) {
+    return 0;
+  }
+
+  return (str[0] == '"' && str[len - 1] == '"') ||
+         (str[0] == '\'' && str[len - 1] == '\'');
```

Comment:
utils_cmds.c currently only supports double quotes. I think we should do the same here.

Reply:
Done.

Reply:
@RafaelMarinheiro , don't publish code which you don't use and don't checked.
... And don’t trust anyone.